spec/models/stats/stats_report_spec.rb

```diff
@@ -167,6 +167,64 @@
     end
   end
 
+  describe '#document#path' do
+    let(:report) { create :stats_report, document_file_name: 'test_file.csv' }
+
+    before do
+      stub_const 'REPORTS_STORAGE_PATH', 'reports/:filename'
+    end
+
+    context 'without an Active Storage attachment' do
+      it 'has a path based on the filename' do
+        expect(report.document.path).to eq 'reports/test_file.csv'
+      end
+    end
+
+    context 'with an Active Storage attachment in disk storage' do
+      require 'active_storage/service/disk_service'
+
+      before do
+        ActiveStorage::Attachment.connection.execute(<<~SQL)
```

Comment:
[as above](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3816/files#r596672810)